src/test/java/com/snack/news/controller/NewsControllerTest.java

```diff
@@ -43,8 +44,8 @@
 
 	private MockMvc mockMvc;
 
-	@Before
-	public void setup() {
+	@BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4764524)